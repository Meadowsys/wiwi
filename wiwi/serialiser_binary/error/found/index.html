<!DOCTYPE html>
<html lang="en">
<head>
    <meta http-equiv="refresh" content="0;URL=../../../../wiwi/serialiser_binary/found/index.html">
    <title>Redirection</title>
</head>
<body>
    <p>Redirecting to <a href="../../../../wiwi/serialiser_binary/found/index.html">../../../../wiwi/serialiser_binary/found/index.html</a>...</p>
    <script>location.replace("../../../../wiwi/serialiser_binary/found/index.html" + location.search + location.hash);</script>
</body>
</html>