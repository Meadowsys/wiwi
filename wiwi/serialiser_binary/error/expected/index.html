<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta name="generator" content="rustdoc"><meta name="description" content="error messages for the `expected` part of errors"><title>wiwi::serialiser_binary::error::expected - Rust</title><script>if(window.location.protocol!=="file:")document.head.insertAdjacentHTML("beforeend","SourceSerif4-Regular-46f98efaafac5295.ttf.woff2,FiraSans-Regular-018c141bf0843ffd.woff2,FiraSans-Medium-8f9a781e4970d388.woff2,SourceCodePro-Regular-562dcc5011b6de7d.ttf.woff2,SourceCodePro-Semibold-d899c5a5c4aeb14a.ttf.woff2".split(",").map(f=>`<link rel="preload" as="font" type="font/woff2" crossorigin href="../../../../static.files/${f}">`).join(""))</script><link rel="stylesheet" href="../../../../static.files/normalize-76eba96aa4d2e634.css"><link rel="stylesheet" href="../../../../static.files/rustdoc-dd39b87e5fcfba68.css"><meta name="rustdoc-vars" data-root-path="../../../../" data-static-root-path="../../../../static.files/" data-current-crate="wiwi" data-themes="meadow" data-resource-suffix="" data-rustdoc-version="1.81.0-nightly (e9e6e2e44 2024-06-28)" data-channel="nightly" data-search-js="search-0fe7219eb170c82e.js" data-settings-js="settings-4313503d2e1961c2.js" ><script src="../../../../static.files/storage-118b08c4c78b968e.js"></script><script defer src="../sidebar-items.js"></script><script defer src="../../../../static.files/main-20a3ad099b048cf2.js"></script><noscript><link rel="stylesheet" href="../../../../static.files/noscript-df360f571f6edeae.css"></noscript><link rel="alternate icon" type="image/png" href="../../../../static.files/favicon-32x32-422f7d1d52889060.png"><link rel="icon" type="image/svg+xml" href="../../../../static.files/favicon-2c020d218678b618.svg"></head><body class="rustdoc mod"><!--[if lte IE 11]><div class="warning">This old browser is unsupported and will most likely display funky things.</div><![endif]--><nav class="mobile-topbar"><button class="sidebar-menu-toggle" title="show sidebar"></button></nav><nav class="sidebar"><div class="sidebar-crate"><h2><a href="../../../../wiwi/index.html">wiwi</a><span class="version">0.10.0-dev</span></h2></div><h2 class="location"><a href="#">Module expected</a></h2><div class="sidebar-elems"><section><ul class="block"><li><a href="#statics">Statics</a></li></ul></section><h2><a href="../index.html">In wiwi::serialiser_binary::error</a></h2></div></nav><div class="sidebar-resizer"></div><main><div class="width-limiter"><rustdoc-search></rustdoc-search><section id="main-content" class="content"><div class="main-heading"><h1>Module <a href="../../../index.html">wiwi</a>::<wbr><a href="../../index.html">serialiser_binary</a>::<wbr><a href="../index.html">error</a>::<wbr><a class="mod" href="#">expected</a><button id="copy-path" title="Copy item path to clipboard">Copy item path</button></h1><span class="out-of-band"><a class="src" href="../../../../src/wiwi/serialiser_binary/error.rs.html#122-205">source</a> · <button id="toggle-all-docs" title="collapse all docs">[<span>&#x2212;</span>]</button></span></div><span class="item-info"><div class="stab portability">Available on <strong>crate feature <code>serialiser-binary-unstable</code></strong> only.</div></span><details class="toggle top-doc" open><summary class="hideme"><span>Expand description</span></summary><div class="docblock"><p>error messages for the <code>expected</code> part of errors</p>
</div></details><h2 id="statics" class="section-header">Statics<a href="#statics" class="anchor">§</a></h2><ul class="item-table"><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_ARRAY.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_ARRAY">DESC_EXPECTED_ARRAY</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_BOOL.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_BOOL">DESC_EXPECTED_BOOL</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_EOF.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_EOF">DESC_EXPECTED_EOF</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_F32.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_F32">DESC_EXPECTED_F32</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_F64.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_F64">DESC_EXPECTED_F64</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I8.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I8">DESC_EXPECTED_I8</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I16.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I16">DESC_EXPECTED_I16</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I24.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I24">DESC_EXPECTED_I24</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I32.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I32">DESC_EXPECTED_I32</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I40.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I40">DESC_EXPECTED_I40</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I48.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I48">DESC_EXPECTED_I48</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I56.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I56">DESC_EXPECTED_I56</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I64.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I64">DESC_EXPECTED_I64</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I72.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I72">DESC_EXPECTED_I72</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I80.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I80">DESC_EXPECTED_I80</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I88.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I88">DESC_EXPECTED_I88</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I96.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I96">DESC_EXPECTED_I96</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I104.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I104">DESC_EXPECTED_I104</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I112.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I112">DESC_EXPECTED_I112</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I120.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I120">DESC_EXPECTED_I120</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_I128.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_I128">DESC_EXPECTED_I128</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_ISIZE.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_ISIZE">DESC_EXPECTED_ISIZE</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_MARKER.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_MARKER">DESC_EXPECTED_MARKER</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_STR.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_STR">DESC_EXPECTED_STR</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U8.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U8">DESC_EXPECTED_U8</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U16.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U16">DESC_EXPECTED_U16</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U24.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U24">DESC_EXPECTED_U24</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U32.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U32">DESC_EXPECTED_U32</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U40.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U40">DESC_EXPECTED_U40</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U48.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U48">DESC_EXPECTED_U48</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U56.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U56">DESC_EXPECTED_U56</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U64.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U64">DESC_EXPECTED_U64</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U72.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U72">DESC_EXPECTED_U72</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U80.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U80">DESC_EXPECTED_U80</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U88.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U88">DESC_EXPECTED_U88</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U96.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U96">DESC_EXPECTED_U96</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U104.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U104">DESC_EXPECTED_U104</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U112.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U112">DESC_EXPECTED_U112</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U120.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U120">DESC_EXPECTED_U120</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_U128.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_U128">DESC_EXPECTED_U128</a></div></li><li><div class="item-name"><a class="static" href="static.DESC_EXPECTED_USIZE.html" title="static wiwi::serialiser_binary::error::expected::DESC_EXPECTED_USIZE">DESC_EXPECTED_USIZE</a></div></li></ul></section></div></main></body></html>